lib/roo_on_rails/sidekiq/queue_latency.rb

```diff
@@ -5,6 +5,39 @@ module Sidekiq
     class QueueLatency
       extend Forwardable
 
+      DEFAULT_LATENCY_VALUES = {
```

Comment:
Yeah but with this approach we ensure that our queues are sorted by increasing permitted latency. Will keep it as is.